db/migrate/20160515171330_create_reported_events.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReportedEvents < ActiveRecord::Migration
+  def change
+    create_table :reported_events do |t|
+      t.references :event
+      t.references :user
+      t.string :reason
```

Comment:
String ma tylko 255 znaków, proponuję to zamienić na text i dać przy walidacji jakieś ograniczenie długości.